### PR TITLE
Game Features & Weekly Salaries & Rankouts CU-2u09kdq

### PR DESCRIPTION
**Pull request by bwpikaard**

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2u09kdq Run Salaries Weekly](https://app.clickup.com/t/2u09kdq)

File: core/src/database/game/feature/feature.enum.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {registerEnumType} from "@nestjs/graphql";
+
+export enum FeatureCode {
+    /**
+     * Automatically run salaries every Monday at noon.
```
Comment (0c370t) [inline core/src/database/game/feature/feature.enum.ts:5]: Pedantic but this is likely going to be configurable

File: core/src/database/game/feature/feature.enum.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+     */
+    AUTO_SALARIES = "AUTO_SALARIES",
+    /**
+     * Automatically run rankouts every Monday at noon. (Should be a dependent of AUTO_SALARIES)
```
Comment (0c370t) [inline core/src/database/game/feature/feature.enum.ts:9]: Pedantic but this is likely going to be configurable

File: core/src/elo-connector/elo-connector.consumer.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        //     const autoSalariesEnabled = await this.gameFeatureService.featureIsEnabled(FeatureCode.AUTO_SALARIES, game.id, organization.id);
+
+        //     if (autoRankoutsEnabled) {
+        //         // TODO: Scope to Org/Game
+        //         await this.eloService.processSalaries(true);
+        //     } else if (autoSalariesEnabled) {
+        //         // TODO: Scope to Org/Game
+        //         await this.eloService.processSalaries(false);
+        //     }
+        // }))));
```
Comment (0c370t) [inline core/src/elo-connector/elo-connector.consumer.ts:62]: Remove commented code

**Review by 0c370t — CHANGES_REQUESTED** [review]

**Review by gankoji — APPROVED** [review]

**Comment by 0c370t** [comment]
Fix merge conflicts

**Pull request merged**